dotCMS/src/main/java/com/dotmarketing/db/commands/UpsertCommand.java

```diff
@@ -0,0 +1,287 @@
+package com.dotmarketing.db.commands;
+
+import com.dotcms.system.SimpleMapAppContext;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.util.SQLUtil;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.DbType;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.db.DbType'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)